[IMP] inventory: automatic config instructions (#9756)
odoo/documentation: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/inventory_valuation_config.rst

Price` and the :guilabel:`Inventory Valuation` (not visible by default) set to :guilabel:`Manual`.

.. image:: inventory_valuation_config/inventory-valuation-fields.png
   :align: center
   :alt: The Costing Method field is located on the Product Categories form.

Alternatively, perpetual (automatic) inventory valuation creates real-time *journal entries* in the
*Accounting* app whenever stock enters or leaves the company's warehouse.

This document is focused on the proper setup of automatic inventory valuation, which is an
integrated valuation method that ensures journal entries in *Accounting* match stock valuation
updates in *Inventory*. For an introduction of inventory valuation in Odoo, refer to the
:doc:`using_inventory_valuation` documentation.

.. warning::
   Switching to automatic inventory valuation **will not** create journal entries for existing
   stock, potentially leading to discrepancies between stock valuation and accounting journals.

   To manually post journal entries for existing stock, navigate to :menuselection:`Inventory app
   --> Reporting --> Valuation` for a list of current stock and its value. Then, :doc:`post a

>    journal entry <../../../../finance/accounting/vendor_bills>` for each existing item.

Author: @toaa-odoo Is this the best accounting document to refer to for readers who are looking to manually post journal entries?

Reviewer: Yeah it is fine

This method implies that the accounting team posts journal entries based on the physical inventory
of the company, and that warehouse employees take the time to count the stock. In Odoo, this method
is reflected inside each product category, where the :guilabel:`Costing Method` field will be set to
`Standard Price` by default.

.. image:: inventory_valuation_config/inventory-valuation-fields.png
   :align: center
   :alt: The Costing Method field is located on the Product Categories form.

> Alternatively, automated inventory valuation is an integrated valuation method that updates the

Reviewer: I'm thinking a note could be added (could be a copy/paste of line #78) here about the need to have the Accounting app installed. As of right now, and considering the user follows the flow in the PR (that is, only having inventory installed - without accounting), they're introduced to the concept of "Automatic Inventory Valuation" without seeing the option in their DB. A note would make it clearer for both:

1) Users without Accounting installed at that point in time, as they would know about this other feature explained to them, but also why it's not available to them in their DB
2) Users with Accounting installed: so far, this seems to pertain only to Inventory users (everything revolves around Inventory) when it is also meant for Accounting users. Maybe a :ref: could be added to line #78 for those looking for automated valuation.

> Alternatively, automated inventory valuation is an integrated valuation method that updates the

Reviewer: ```suggestion
Alternatively, :ref:`automated inventory valuation <line 78>` is an integrated valuation method that updates the
```

> change. If the value in the :guilabel:`Cost` field on a product form is changed manually, Odoo will
> generate a corresponding record in the *Inventory Valuation* report.

Reviewer 2: ```suggestion
change. If the value in the :guilabel:`Cost` field on a product form is changed manually, Odoo
generates a corresponding record in the *Inventory Valuation* report.
```

Author: Manually applied this change on my end (:

.. note::
   It is possible to use different valuation settings for different product categories.

.. _inventory/warehouses_storage/accounting-types:

Types of accounting
===================

With automated inventory valuation set up, the generated journal entries depend on the chosen
accounting mode: *Continental* or *Anglo-Saxon*.

.. tip::

>    Verify the accounting mode by activating the :ref:`developer-mode` and navigating to

Reviewer 2: ```suggestion
   Verify the accounting mode by activating the :ref:`developer-mode`, and navigating to
```
OPTIONAL: added comma for enhanced sentence pacing

   :menuselection:`Accounting app --> Configuration --> Settings`.

>    Then, in the search bar, look for :guilabel:`Anglo-Saxon Accounting`, to see if the feature is

Reviewer 2: ```suggestion
   Then, in the :guilabel:`Search...` bar, look for `Anglo-Saxon Accounting`, to see if the feature is
```

   enabled. If it is **not** enabled, Continental accounting mode is in use.

   .. image:: inventory_valuation_config/anglo-saxon.png
      :align: center
      :alt: Show the Anglo-Saxon accounting mode feature.

In *Anglo-Saxon* accounting, the costs of goods sold (COGS) are reported when products are sold or
delivered. This means that the cost of a good is only recorded as an expense when a customer is
invoiced for a product. So for **manual** valuation method, set the :guilabel:`Expense Account` to

> `Stock Valuation` for the current asset type; for **automatic** valuation method, set the the

Reviewer: ```suggestion
`Stock Valuation` for the current asset type; for **automatic** valuation method, set the
```

> delivered. This means that the cost of a good is only recorded as an expense when a customer is
> invoiced for a product. So for **manual** valuation method, set the :guilabel:`Expense Account` to
> `Stock Valuation` for the current asset type; for **automatic** valuation method, set the
> :guilabel:`Expense Account` to an *Expenses* or a *Cost of Revenue* type (e.g. `Cost of Production`,
> `Cost of Goods Sold`, etc.).

Reviewer 2: ```suggestion
delivered. This means the cost of a good is only recorded as an expense when a customer is
invoiced for a product.

So, for **manual** valuation method, set the :guilabel:`Expense Account` to *Stock Valuation* for
the current asset type; for **automatic** valuation method, set the :guilabel:`Expense Account` to
an *Expenses* or a *Cost of Revenue* type (e.g. *Cost of Production*, *Cost of Goods Sold*, etc.).
```
suggesting the use of italics here because they are proper setting options that can be chosen BUT aren't being shown directly on the page at this point. However, if you disagree, or if they are actually options that are manually typed in by the user, feel free to keep it as is
